src/private/meeting.ts

```diff
@@ -0,0 +1,97 @@
+import { ensureInitialized, sendMessageRequestToParent } from '../internal/internalAPIs';
+import { GlobalVars } from '../internal/globalVars';
+import { MeetingRoomCapability, MeetingRoomInfo, MeetingRoomState } from './interfaces';
+
+export namespace meeting {
+  let meetingRoomCapabilitiesUpdateHandler: (capabilities: MeetingRoomCapability) => void;
+  GlobalVars.handlers['meeting.meetingRoomCapabilitiesUpdate'] = handleMeetingRoomCapabilitiesUpdate;
+  let meetingRoomStatesUpdateHandler: (states: MeetingRoomState) => void;
+  GlobalVars.handlers['meeting.meetingRoomStatesUpdate'] = handleMeetingRoomStatesUpdate;
+
+  /**
+   * @private
+   * Hide from docs
+   *
+   * Fetch the meeting room info that paired with current client.
+   * @param callback Callback to invoke when the meeting room info is fetched.
+   */
+  export function getPairedMeetingRoomInfo(callback: (meetingRoomInfo: MeetingRoomInfo) => void): void {
+    ensureInitialized();
+    const messageId = sendMessageRequestToParent('meeting.getPairedMeetingRoomInfo');
+    GlobalVars.callbacks[messageId] = callback;
+  }
+
+  /**
+   * @private
+   * Hide from docs
+   *
+   * Send a command to paired meeting room.
+   * @param commandName The command name.
+   * @param callback Callback to invoke when the command response returns.
+   */
+  export function sendCommandToPairedMeetingRoom(
+    commandName: string,
+    callback: (errorCode: number, message?: string) => void,
```

Comment:
Please use the SdkError to send back error information

Reply:
OK. Make sense.